lib/elixir_internal_certificate_web/controllers/user_registration_controller.ex

```diff
@@ -0,0 +1,24 @@
+defmodule ElixirInternalCertificateWeb.UserRegistrationController do
+  use ElixirInternalCertificateWeb, :controller
+
+  alias ElixirInternalCertificate.Accounts
+  alias ElixirInternalCertificate.Accounts.User
+  alias ElixirInternalCertificateWeb.UserAuth
+
+  def new(conn, _params) do
+    changeset = Accounts.change_user_registration(%User{})
+    render(conn, "new.html", changeset: changeset)
```

Comment:
Lets add a new line for better redability
```suggestion

    render(conn, "new.html", changeset: changeset)
```